### PR TITLE
Support stripe_bank_account_token in User.exchange_token

### PR DESCRIPTION
**Pull request by be9**
Based on #96

### CODE REVIEW

**Comment by be9** [comment]
@michaelckelly @rbuckheit Let's make it the final PR before 2.0.0 release!

File: README.md [hunk 1]
```diff
@@ -89,6 +89,9 @@ With more options:
 user2 = Plaid::User.exchange_token('public_token', account_id: '...', product: :auth)
 ```
 
+If you specify the `account_id` param, the returned user instance will have the
+`stripe_bank_account_token` attribute set.
+
```
Comment (rbuckheit) [inline README.md]: I admittedly don't have great context on how stripe works, what is `account_id` and how does it relate to `stripe_bank_account_token`?

Comment (be9) [inline README.md]: It's described here: https://plaid.com/docs/link/stripe/#step-4-write-server-side-handler

@michaelckelly anything to add?

Comment (michaelckelly) [inline README.md]: > @michaelckelly anything to add?

Can we add a link to that section of the docs within the README?

**Comment by michaelckelly** [comment]
LGTM other than the comment regarding the README, @be9

**Comment by be9** [comment]
@michaelckelly Added the link.

**Comment by michaelckelly** [comment]
LGTM @be9 - I may follow up this PR with some additions to the README, but this is good as is.

**Pull request merged**